allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_nwb_session_api.py

```diff
@@ -55,6 +57,22 @@ def __init__(self,
         self.additional_unit_metrics = additional_unit_metrics
         self.external_channel_columns = external_channel_columns
 
+        self._check_nwbfile_version()
+
+    def _check_nwbfile_version(self):
+        if hasattr(self, "path") and self.path:
+            with h5py.File(self.path, 'r') as nwb_file:
+                nwb_version = str(nwb_file.attrs["nwb_version"]).split(".")
+
+            if tuple(nwb_version) < ("2", "2", "2"):
```

Comment:
Can nwb versions ever have patch components with alphabetical values? How do those compare here?

Reply:
Just saw your test case for this - nice!